docs/tutorials/quantum_data.ipynb

```diff
@@ -787,14 +782,16 @@
    "source": [
     "Now you have everything you need to re-label the dataset! Now you can consult with the flowchart to better understand how to maximize performance seperation when re-labeling the dataset:\n",
     "\n",
-    "<img src=\"./images/quantum_data_1.png\">\n",
+    "<img src=\"https://github.com/tensorflow/quantum/blob/master/docs/tutorials/images/quantum_data_1.png?raw=1\">\n",
```

Comment:
This change was made by our website and should be kept as just `"<img src=\"./images/quantum_data_1.png\">\n",`